test/Microsoft.ML.TestFramework/Learners.cs

```diff
@@ -59,14 +59,14 @@ static TestLearnersBase()
         // New.
         public static PredictorAndArgs binarySdca = new PredictorAndArgs
         {
-            Trainer = new SubComponent("SDCA", "maxIterations=5 checkFreq=9 nt=1"),
+            Trainer = new SubComponent("SDCALR", "maxIterations=5 checkFreq=9 nt=1"),
```

Comment:
>SDCALR [](start = 40, length = 6)

if you want to test it, add new one, don't change already existing!

Reply:
No. Old SDCA without specifying loss function is equivalent to the new SDCALR.

---
In reply to: [256212199](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256212199) [](ancestors = 256212199)

Reply:
Why do you call it SDCALR then? Can you still call it SDCA?

---
In reply to: [256219447](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256219447) [](ancestors = 256219447,256212199)

Reply:
The original SDCA is for SVM, not LR. I'd like to following classical convention in academic community. Is it ok to you?

---
In reply to: [256220043](https://github.com/dotnet/machinelearning/pull/2506#discussion_r256220043) [](ancestors = 256220043,256219447,256212199)